README.md

```diff
@@ -1,202 +1,77 @@
-# IoT Reference Integration on STM32U5 Arm Cortex-M33 MCU
+# Getting Started Guide for STM32U5 IoT Discovery Kit with AWS
 
+## Overview
 
-### Perfroming Over-the-air (OTA) Firmware Update for [B-U585I-IOT02A discovery kit](https://www.st.com/en/evaluation-tools/b-u585i-iot02a.html)
+This project demonstrates how to integrate modular FreeRTOS software with hardware enforced security to create secure and updatable cloud connected applications. The project is pre-configured to run on the STM32U585 IoT Discovery Kit and connect to AWS.
 
-The project shows an IoT reference implementation of how to integrate FreeRTOS libraries on STM32U5 platform to perform OTA update with AWS IoT using both trustzone and non-trustzone hardware capablities. The demo runs FreeRTOS OTA agent as one of the RTOS tasks in background, which waits for OTA updates from cloud. The non-trustzone version of the demo leverages internal flash memory's dual bank. The total 2MB internal flash is split into two banks of 1MB each. The main firmware is running on one bank, while downloading new firmware image into another bank. Thus, the total firmware image size should not exceed 1MB.
+The STM32U585 IoT Discovery kit comes with 2048 KB of Flash memory, 786 kB of RAM and is based on Arm Cortex®-M33.
 
-Please follow the below steps to bring up the demo project and perform OTA update:
+The STM32U5 IoT Discovery Kit is equipped with a Wi-Fi and Bluetooth module, microphones, a temperature and humidity sensor, a magnetometer, an accelerometer and gyroscope, a pressure sensor, as well as Time-of-Flight (ToF) and gesture-detection sensors.
 
```

Comment:
Suggest to add also: "The demo connects to AWS IoT over Wi-Fi. It then uses the coreMQTT-Agent library to enable multiple concurrent tasks to share the connection. Those tasks publish sensor data, and demonstrate use of the Device Shadow and Device Defender AWS IoT services. There are two projects included, with and without Trust Firmware-M(https://www.trustedfirmware.org/projects/tf-m/) enabled.